doc/services/device_mgmt/smp_groups/smp_group_63.rst

```diff
@@ -0,0 +1,92 @@
+.. _mcumgr_smp_group_63:
+
+Zephyr Management Group
+#######################
+
+Zephyr management group defines the following commands:
+
+.. table::
+    :align: center
+
+    +----------------+------------------------------+
+    | ``Command ID`` | Command description          |
+    +================+==============================+
+    | ``0``          | Erase storage                |
+    +----------------+------------------------------+
+
+Erase storage command
+*********************
+
+Erase storage command allows clearing the ``storage_partition`` flash partition on a device,
+generally this is used when switching to a new application build if the application uses storage
+that should be cleared (application dependent).
+
+Erase storage request
+=====================
+
+Erase storage request header fields:
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``2``  | ``63``       | ``0``          |
+    +--------+--------------+----------------+
+
+The command sends sends empty CBOR map as data.
+
+Erase storage response
+======================
+
+Read setting response header fields:
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``3``  | ``63``       | ``0``          |
+    +--------+--------------+----------------+
+
+The command sends an empty CBOR map as data if successful. In case of error the CBOR data takes
+the form:
+
+.. tabs::
+
+   .. group-tab:: SMP version 2
+
+      .. code-block:: none
+
+          {
+              (str)"err" : {
+                  (str)"group"    : (uint)
+                  (str)"rc"       : (uint)
+              }
+          }
```

Comment:
no obligation, but just wondering if you'd like to maybe use CDDL notation? It could benefit from syntax highlighting too

```suggestion
      .. code-block:: cddl

          resp = {
             "err" : {
                "group"    : uint
                "rc"       : uint
             }
          }
```

This might not make a big difference for this particular example, but for other instances where you want to document optional fields etc, this could help make things more readable and expose a standard schema.

https://www.rfc-editor.org/rfc/rfc8610.html
https://github.com/cose-wg/Examples/blob/master/examples.cddl

![image](https://github.com/zephyrproject-rtos/zephyr/assets/128251/0f46b6ff-e8cb-4935-b3b8-64e9bb4a106d)

Reply:
Don't know what the above would look like but the types are important for users wanting to decode the fields

Reply:
but the types are there in my suggested change above, no?

![image](https://github.com/zephyrproject-rtos/zephyr/assets/128251/a6d24902-9b68-4e20-b2aa-6d56665eaef4)

Reply:
group and rc being strings I mean, not their values, the key types themselves